app/src/main/java/com/kickstarter/ui/adapters/ActivityFeedAdapter.java

```diff
@@ -15,25 +16,31 @@
 import com.kickstarter.ui.viewholders.ProjectStateChangedPositiveViewHolder;
 import com.kickstarter.ui.viewholders.ProjectStateChangedViewHolder;
 import com.kickstarter.ui.viewholders.ProjectUpdateViewHolder;
+import com.kickstarter.ui.viewholders.UnansweredSurveyHeaderViewHolder;
+import com.kickstarter.ui.viewholders.UnansweredSurveyViewHolder;
 
 import java.util.Collections;
 import java.util.List;
 
 public final class ActivityFeedAdapter extends KSAdapter {
   private static final int SECTION_LOGGED_IN_EMPTY_VIEW = 0;
   private static final int SECTION_LOGGED_OUT_EMPTY_VIEW = 1;
-  private static final int SECTION_ACTIVITIES_VIEW = 2;
+  private static final int SECTION_SURVEYS_HEADER_VIEW = 2;
+  private static final int SECTION_SURVEYS_VIEW = 3;
+  private static final int SECTION_ACTIVITIES_VIEW = 4;
 
   private final Delegate delegate;
 
   public interface Delegate extends FriendBackingViewHolder.Delegate, ProjectStateChangedPositiveViewHolder.Delegate,
-    ProjectStateChangedViewHolder.Delegate, ProjectUpdateViewHolder.Delegate, EmptyActivityFeedViewHolder.Delegate {}
+    ProjectStateChangedViewHolder.Delegate, ProjectUpdateViewHolder.Delegate, EmptyActivityFeedViewHolder.Delegate, UnansweredSurveyViewHolder.Delegate {}
 
   public ActivityFeedAdapter(final @NonNull Delegate delegate) {
     this.delegate = delegate;
 
     insertSection(SECTION_LOGGED_IN_EMPTY_VIEW, Collections.emptyList());
     insertSection(SECTION_LOGGED_OUT_EMPTY_VIEW, Collections.emptyList());
+    insertSection(SECTION_SURVEYS_HEADER_VIEW, Collections.singletonList(null));
```

Comment:
I think this is the reason you are seeing extra padding at the top. you have inserted a blank row for the header view, and even when survey count is zero it will not get cleared because of the `if` gating.

there are a few ways you could fix it, but perhaps write some tests first for the expectations and then we can discuss refactorings to fix it.